server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -275,6 +277,8 @@ public void execute(final String[] args) {
     } catch (Exception e) {
       log.error("{}", e.getMessage(), e);
       System.exit(3);
+    } finally {
+      SingletonManager.setMode(Mode.CLOSED);
```

Comment:
> Does this prevent this code from being called from a process that might have other Accumulo client Connectors?

Yes I think it would.  I don't think this situation happens currently.  If it ever did in the future, a clear error message would occur making it easy to track down.  Since this not public API, its always easy to change this code later if the hypothetical situation does happen.